packages/label-sync/package-lock.json

```diff
@@ -34,130 +34,6 @@
 				"lazy-ass": "1.6.0"
 			}
 		},
-		"@bcoe/v8-coverage": {
-			"version": "0.2.3",
-			"resolved": "https://registry.npmjs.org/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz",
```

Comment:
as with the blunderbuss changes, you shouldn't need to make any updates to these files, I think you can use the:

`git checkout origin/master [filename]`

Approach to revert any files that changed in a folder other than `googleapis/generate-bot`.